index-import.cpp

```diff
@@ -208,18 +214,18 @@ static bool cloneRecords(StringRef recordsDirectory,
     SmallString<128> outputPath{inputPath};
     path::replace_path_prefix(outputPath, inputIndexPath, outputIndexPath);
 
-    std::error_code failed;
+    bool failed = false;
     if (status->type() == fs::file_type::directory_file) {
       if (not fs::exists(outputPath)) {
```

Comment:
... you can avoid this. stat and use the pattern in my other PR

Reply:
I was going to wait rebase/merge on top of your change.

Reply:
SGTM